Experiment result text (#83)
chhoumann/p8: report/src/system_evaluation/results.tex

\section{System evaluation}\label{sec:experiment_results}
%describe in terms of criteria for using bluetooth. `'
The collection of measurements using the app is carried out in the same room as the map is created. 
However, while the room was unoccupied during the creation of the map, this was not the case for the 
second part of the experiment. 
During this experiment, computers and phone where placed near the tables of the room, with several of them having either only wifi or Bluetooth and Wifi enabled. 

> In the common area outside the room, two people were tasked with moving around, and a laptop with Bluetooth and Wifi enabled was placed on one of the tables.

Reviewer: ```suggestion
In the common area outside the room, two people were tasked with moving around, and a laptop with Bluetooth and WiFi enabled was placed on one of the tables.
```

Reviewer: This sounds unstructured. What are we trying to say here and what is the purpose?

    3                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10}  & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{9/10}  & 10/10                  \\ \hline
    4                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{8/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10}  & \multicolumn{1}{l|}{9/10} & \multicolumn{1}{l|}{8/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{10/10} & 10/10                  \\ \hline
    \end{tabular}}
    \caption{Evaluation results from Samsung Galaxy A53 showing the number of correct classifications out of a possible ten for each k-value}
    \label{lst:resultsPhone}
\end{table}
All of this is done to create a more realistic environment, resembling that of a meeting room in use during working hours. 

> Results were gathered in the location as shown on Figure \ref{fig:experiment_room} by making 10 classifications using the app described in Section \ref{todo} \todo{refer to the app} on a Samsung Galaxy A53 phone.

Reviewer: Cant commit without fixing this todo

During the classification, k-values of 2, 3, and 4 were used. 
The results of the experiment can be seen in Table \ref{lst:resultsPhone}.

As expected, a lower k-value results in a larger uncertainty in the classification.
However, it is worth noticing that for measuring points \textit{B} and \textit{J}  (see Figure \ref{fig:experiment_room}) a large uncertainty in classification is observed.  

> We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 

Reviewer: ```suggestion
We suspect that this is a consequence of the glass pane next to the door having a much lower absorption of radio frequencies than the door and the wall next to it. 
```

> This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as inside and some as outside, therefore giving uncertain classifications.

Reviewer: ```suggestion
This low absorption results in the two data points next to the pane having similar distances to both map values labelled as inside and some as outside, therefore giving uncertain classifications.
```

\section{System evaluation}\label{sec:experiment_results}
%describe in terms of criteria for using bluetooth. `'

> Having created a map that can be used to classify received measurements, we can now begin to examine wether these classifications are accurate. 

Reviewer: ```suggestion
Having created a map that can be used to classify received measurements, we can now begin to examine whether these classifications are accurate. 
```

The measurements are collected using the implemented functionality described in Section \ref{sec:knn_implementation}.

> However, while the room was unoccupied during the creation of the map, this was not the case for this part of the experiment. 

Reviewer: Maybe we should detail why it did not matter that the room had people in it during the testing. 

Additionally, unoccupied/occupied may not be the right words for this, since we are implying that we are making tests in a room that other people have "booked".

> Computers and phones where placed near the tables of the room, with several of them having either only WiFi or both Bluetooth and WiFi enabled. 

Reviewer: ```suggestion
Computers and phones were placed near the tables of the room, with several of them having either only WiFi or both Bluetooth and WiFi enabled. 
```

> In the common area outside the room, two people were tasked with moving around, and a laptop with Bluetooth and WiFi enabled was placed on one of the tables.

Reviewer: Why are we saying two people were tasked with this? Is this relevant? If so we should also specify how many people did measurements inside the room.

Author: Aebeloe requested a really detailed explanation of how the experiment is performed. This is due to the fluctuations caused by even small changes to the physical environment.

        2                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{8/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10}  & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{0/10} & \multicolumn{1}{l|}{4/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{7/10} & 8/10                   \\ \hline
        3                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{8/10} & 8/10                   \\ \hline
        4                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{9/10} & 10/10                  \\ \hline
        \end{tabular}}

>         \caption{Evaluation results from Lenovo TabM10 FHD Plus showing the number of correct classifications out of a possible ten for each k-value}

Reviewer: ```suggestion
        \caption{Evaluation results from Lenovo TabM10 FHD Plus showing the number of correct classifications out of a possible ten for each $k$-value}
```

        \label{lst:resultsTablet}
        \end{table}

> To create an environment that more accurately represents a meeting room in use, several devices were used for testing. 

Reviewer: I added this

> The method for collecting results from these tests mirrored previous ones, by making 10 classifications on both a Samsung Galaxy A53, a Samsung Galaxy S20 phone, and a Lenovo TabM10 FHD Plus tablet. 

Reviewer: I added this

> The method for collecting results from these tests mirrored previous ones, by making 10 classifications on both a Samsung Galaxy A53, a Samsung Galaxy S20 phone, and a Lenovo TabM10 FHD Plus tablet. 

Reviewer: ```suggestion
The method for collecting results from these tests mirrored previous ones, by making $10$ classifications on both a Samsung Galaxy A53, a Samsung Galaxy S20 phone, and a Lenovo TabM10 FHD Plus tablet. 
```

The classification results were gathered in the locations shown on Figure \ref{fig:experiment_room}

> During the experiment, k-values of 2, 3, and 4 were used.  

Reviewer: ```suggestion
During the experiment, k-values of $2$, $3$, and $4$ were used.  
```

> During the experiment, k-values of 2, 3, and 4 were used.  

Reviewer: ```suggestion
During the experiment, $k$-values of 2, 3, and 4 were used.  
```

The results of the experiment can be seen in Tables \ref{lst:resultsPhone}, \ref{lst:resultsPhoneS20}, and  \ref{lst:resultsTablet}.

> Table \ref{lst:resultsPhone_precision} summerizes the results of the experimentin terms of percentage of true positive classifications. 

Reviewer: ```suggestion
Table \ref{lst:resultsPhone_precision} summarizes the experiment's results, indicating the percentage of true positive classifications. 
```

\begin{table}[H]
    \centering
    \begin{tabular}{|l|ll|}
    \hline
            & \multicolumn{2}{l|}{Accuracy}                      \\ \hline
    K-Value & \multicolumn{1}{l|}{Indoor}          & Outside      \\ \hline
    2       & \multicolumn{1}{l|}{200/210 (95.24\%)} & 68/150 (45.33\%) \\ \hline
    3       & \multicolumn{1}{l|}{200/210 (95.24\%)} & 96/150 (64.00\%) \\ \hline
    4       & \multicolumn{1}{l|}{205/210 (97.62\%)} & 118/150 (78.67\%) \\ \hline   
    \end{tabular}

>     \caption{Accuracy of the classifications for different k-values}

Reviewer: ```suggestion
    \caption{Accuracy of the classifications for different $k$-values}
```

    \label{lst:resultsPhone_precision}
\end{table}

> It is seen that the system can determine that the occupant is inside the room with high accuracy even when a small k-value is selected. The approach is not as accurate when detecting that the occupant is outside the room when small values of k are selected.  

Reviewer: ```suggestion
As can be seen in table \ref{lst:resultsPhone_precision}, the results show that the system can determine that the occupant is inside the room with high accuracy, even when a small k-value is selected. The approach is not as accurate when detecting that the occupant is outside the room, when small values of k are selected.  
```

> It is seen that the system can determine that the occupant is inside the room with high accuracy even when a small k-value is selected. The approach is not as accurate when detecting that the occupant is outside the room when small values of k are selected.  

Reviewer: ```suggestion
It is seen that the system can determine that the occupant is inside the room with high accuracy even when a small k-value is selected. The approach is not as accurate when detecting that the occupant is outside the room when small values of $k$ are selected.  
```
